config.json

```diff
@@ -75,7 +75,12 @@
       "core": false,
       "difficulty": 3,
       "slug": "acronym",
-      "topics": null,
+      "topics": [
+        "loops",
+        "strings",
+        "searching",
```

Comment:
`loops`, `strings` and `parsing` all seem like great suggestions :) I'm not sure about `searching` though. How would you say `searching` relates to converting a string to its acronym?

Reply:
@FridaTveit I think we need to do a linear search through the string.

Reply:
Okay, that makes sense :) Happy to merge it then, thanks for explaining! :)